api/proto/teleport/workloadidentity/v1/sigstore.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2025 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.workloadidentity.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/workloadidentity/v1;workloadidentityv1";
+
+// Sigstore bundle and simple signing envelope discovered by `tbot` and sent to
+// the server for verification.
+message SigstoreVerificationPayload {
+  // Sigstore bundle serialized in the protobuf encoding.
```

Comment:
Out of curiosity - is there a reason to not directly embed the message and to have it marshalled as bytes instead?

Reply:
Great question! There were a couple of reasons I went for this approach instead:

1. Sigstore's protobuf specs aren't published in the Buf Schema Registry, so we'd need to vendor their files and it wasn't immediately clear how best to do this
2. More significantly: the `sigstore-go` library uses the `protoc`-generated Go code so I think we'd need to convert between ours and their structs

More that happy to change the approach though! It would feel cleaner.